contrib/clojure-package/src/org/apache/clojure_mxnet/util.clj

```diff
@@ -218,6 +218,16 @@
     (throw (ex-info error-msg
                     (s/explain-data spec value)))))
 
+(s/def ::non-empty-seq sequential?)
+(defn to-array-nd
+  "Converts any N-D sequential structure to an array
+   with the same dimensions."
+  [s]
+  (validate! ::non-empty-seq s "Invalid N-D sequence")
+  (if (sequential? (first s))
+    (to-array (map to-array-nd s))
```

Comment:
Should we call `mapv`? So that thunks for the lazy sequence are not built up?

Reply:
done